lib/cc/analyzer/engine.rb

```diff
@@ -34,7 +34,7 @@ def run(stdout_io, container_listener)
         )
 
         container.on_output("\0") do |raw_output|
-          CLI.debug "#{name} engine output: #{raw_output.inspect}"
+          CLI.debug("#{name} engine output: #{raw_output.gsub("\n", "\\n").gsub("\t", "\\t")}")
```

Comment:
> You could wrap it in quotes to make that more evident if that would help

I was considering that, but I think we need a delimiter other than quotes, since pretty much all engine output will contain quotes.